### PR TITLE
Added Opus TedTalks

### PR DESCRIPTION
**Pull request by rkc007**
Dataset : http://opus.nlpl.eu/TedTalks.php

### CODE REVIEW

**Comment by rkc007** [comment]
@lhoestq I saw some common changes you made on the other PR's (Similar Opus Datasets). I fixed those changes here. Can you please review it once ? 
Thanks.

File: datasets/opus_tedtalks/dataset_infos.json [hunk 1]
```diff
@@ -0,0 +1 @@
...n  date = {23-25},\n  address = {Istanbul, Turkey},\n  editor = {Nicoletta Calzolari (Conference Chair) and Khalid Choukri and Thierry Declerck and Mehmet Ugur Dogan and Bente Maegaard and Joseph Mariani and Jan Odijk and Stelios Piperidis},\n  publisher = {European Language Resources Association (ELRA)},\n  isbn = {978-2-9517408-7-7},\n  language = {english}\n }\n", "homepage": "http://opus.nlpl.eu/TedTalks.php", "license": "", "features": {"id": {"dtype": "string", "id": null, "_type": "Value"}, "translation": {"languages": ["en", "hr"], "id": null, "_type": "Translation"}}, "post_processed": null, "supervised_keys": null, "builder_name": "ted_talks", "config_name": "en-hr", "version": {"version_str": "1.0.0", "description": null, "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 15249417, "num_examples": 86348, "dataset_name": "ted_talks"}}, "download_checksums": {"https://object.pouta.csc.fi/OPUS-TedTalks/v1/moses/en-hr.txt.zip": {"num_bytes": 5639306, "checksum": "25f0162632fbd7dcad896b559c82cb12e1d682534213cf78c6d8318145406706"}}, "download_size": 5639306, "post_processing_size": null, "dataset_size": 15249417, "size_in_bytes": 20888723}}
```
Comment (lhoestq) [inline datasets/opus_tedtalks/dataset_infos.json:1]:
```suggestion
{"en-hr": {"description": "This is a Croatian-English parallel corpus of transcribed and translated TED talks, originally extracted from https://wit3.fbk.eu. The corpus is compiled by \u017deljko Agi\u0107 and is taken from http://lt.ffzg.hr/zagic provided under the CC-BY-NC-SA license.\n2 languages, total number of files: 2\ntotal number of tokens: 2.81M\ntotal number of sentence fragments: 0.17M\n", "citation": "@InProceedings{TIEDEMANN12.463,\n  author = {J\ufffdrg Tiedemann},\n  title = {Parallel Data, Tools and Interfaces in OPUS},\n  booktitle = {Proceedings of the Eight International Conference on Language Resources and Evaluation (LREC'12)},\n  year = {2012},\n  month = {may},\n  date = {23-25},\n  address = {Istanbul, Turkey},\n  editor = {Nicoletta Calzolari (Conference Chair) and Khalid Choukri and Thierry Declerck and Mehmet Ugur Dogan and Bente Maegaard and Joseph Mariani and Jan Odijk and Stelios Piperidis},\n  publisher = {European Language Resources Association (ELRA)},\n  isbn = {978-2-9517408-7-7},\n  language = {english}\n }\n", "homepage": "http://opus.nlpl.eu/TedTalks.php", "license": "", "features": {"id": {"dtype": "string", "id": null, "_type": "Value"}, "translation": {"languages": ["en", "hr"], "id": null, "_type": "Translation"}}, "post_processed": null, "supervised_keys": null, "builder_name": "opus_tedtalks", "config_name": "en-hr", "version": {"version_str": "1.0.0", "description": null, "major": 1, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 15249417, "num_examples": 86348, "dataset_name": "opus_tedtalks"}}, "download_checksums": {"https://object.pouta.csc.fi/OPUS-TedTalks/v1/moses/en-hr.txt.zip": {"num_bytes": 5639306, "checksum": "25f0162632fbd7dcad896b559c82cb12e1d682534213cf78c6d8318145406706"}}, "download_size": 5639306, "post_processing_size": null, "dataset_size": 15249417, "size_in_bytes": 20888723}}
```

`ted_talks`-> `opus_tedtalks`

File: datasets/opus_tedtalks/opus_tedtalks.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        super().__init__(
+            *args,
+            name=f"{lang1}-{lang2}",
+            **kwargs,
+        )
+        self.lang1 = lang1
+        self.lang2 = lang2
+
+
+class OpusTedTalks(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/opus_tedtalks/opus_tedtalks.py:64]:
```suggestion
class OpusTedtalks(datasets.GeneratorBasedBuilder):
```
this must be the camel case version of the dataset script name `opus_tedtalks`

**Review by lhoestq — APPROVED** [review]
thanks !

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**